src/rollback_to_stable/rts_api.c

```diff
@@ -115,6 +115,10 @@ __rollback_to_stable_int(WT_SESSION_IMPL *session, bool no_ckpt)
     WT_ASSERT_SPINLOCK_OWNED(session, &conn->checkpoint_lock);
     WT_ASSERT_SPINLOCK_OWNED(session, &conn->schema_lock);
 
+    /* Check this nice and early, before we do any prep work. */
```

Comment:
Should the check be here? This disables rts for all the situations. I thought we only want to disable it for recovery case. I think it is possible for a caller to call session->rts in the disaggregated storage. It will rollback any unstable updates on the update chain.

Reply:
Done - see https://github.com/wiredtiger/wiredtiger/pull/11413#issuecomment-2588896404.